source/whats-new.txt

```diff
@@ -74,6 +74,28 @@ What's New in 4.11
 
 .. New features of the 4.11 driver release include:
 
+- The following network address-related methods are deprecated and will be removed
+  in v5.0:
+
+  - The `ServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/ServerAddress.html>`__
+    methods ``getSocketAddress()`` and ``getSocketAddresses()``.
+    
+    Instead of ``getSocketAddress()``, use the ``getByName()`` method.
+    ``getByName()`` is an instance method of the ``InetAddress`` class in the
+    ``java.net`` package.
```

Comment:
Suggestion:
I think it would be more concise to represent the class with its package.


```suggestion
    Instead of ``getSocketAddress()``, use the ``getByName()`` instance
    method of ``java.net.InetAddress``.
```

Applies to all instances.